governance-exercises/VoteWizard_poll_exercise_02/README.md

```diff
@@ -0,0 +1,53 @@
+# VoteWizard Polling Exericse #2
+
+## Assignment #1: Determine what course of action you would take as a Governance Facilitator and communicate it clearly
+
+### The course of action I would take as a Governance Facilitator:
+
+1. I'd reply to Clive's Reply #1, explaining to him and the community the criteria I used to determine that the materials shared by Diane constitute reasonable suspicion (or significant suspicion). I would emphasize that the voters will ultimately make the decision.
+
+2. I'd respond to Diane's Reply #2, explaining how the bounty process will be executed if the poll is approved.
+
+3. I plan to include the poll in the Weekly Governance Cycle beginning on Monday, July 28th.
+
+4. Subsequently, I would draft the governance poll.
+
+5. I'd run the text through Grammarly to eliminate any spelling or grammar errors.
+
+6. I'd review the draft poll against the "poll creation checklist" to ensure it is following the guidelines. Then, I would create a PR to push changes to the main branch repo.
+
+7. I would follow the "poll submission checklist" to submit the finalized polls. For tags, the poll's impact is "low-impact", I'd also include tags like 'delegates', 'misc-governance', and 'endgame'.
+
+8. After verifying the active status of all expected polls, I would communicate their availability on the #governance channel in the official MakerDAO discord.
+
+## Assignment #2: Draft any forum responses required or implied by your course of action
+
+### Response to Clive's Reply #1:
+
+Clive,
+
+I appreciate your perspective on this matter. The decision to propose a poll is based on specific criteria, indicating a "significant suspicion":
+
+[Criteria to consider this significant suspicion]
+
+The Atlas is explicit:
+
+> If there’s clear evidence or significant suspicion that the operational security of an AD has been compromised, or that they have failed to follow best practice or otherwise made operational security errors, FacilitatorDAOs must immediately derecognize the AD.
+
+The voters will ultimately decide whether the provided materials sufficiently indicate a compromise of your operational security.
+
+
+### Response to Diane's Reply #2:
+
+Diane, according to the Atlas, if the poll is approved:
```

Comment:
As noted in the poll, I don't think it makes sense to include the bounty payment in the poll decision. Would be better to handle it privately.